packages/devtools_app/lib/src/shared/diagnostics/references.dart

```diff
@@ -68,14 +68,15 @@ Future<void> addChildReferences(
           _refreshStaticSelection(ref.heapSelection, ref.instanceRef);
 
       variable.addAllChildren([
-        DartObjectNode.references(
-          'live (objects currently alive in the application)',
-          ObjectReferences.copyWith(
-            ref,
-            refNodeType: RefNodeType.liveRefRoot,
-            heapSelection: selection,
+        if (ref.instanceRef != null)
```

Comment:
why do we check that `ref.instanceRef != null` here but not below for static references? similar question is why are the if checks difference for static and live?

Reply:
Because we use different mechanisms to show static and live references.
It can be that we can show static, but not live and vise versa.